ts_web_app/src/BUILD.bazel

```diff
@@ -0,0 +1,90 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@aspect_rules_swc//swc:defs.bzl", "swc_transpiler")
+load("@bazel_skylib//lib:partial.bzl", "partial")
+load("@aspect_rules_esbuild//esbuild:defs.bzl", "esbuild")
+load("@aspect_rules_webpack//webpack:defs.bzl", "webpack_bundle", "webpack_dev_server")
+load("@npm//:live-server/package_json.bzl", server = "bin")
+
+ts_project(
```

Comment:
I keep writing docs about how a macro is the perfect way to create your own build system. Would this example be suitable to split into a macro with all the building blocks, exposing a few attributes that make sense from BUILD?